collect_app/src/main/java/org/odk/collect/android/formentry/saving/FormSaveViewModel.java

```diff
@@ -372,6 +385,14 @@ private void clearMediaFiles() {
         recentFiles.clear();
     }
 
+    public LiveData<String> getAnswerFileError() {
```

Comment:
I wanted this error to be general as I think we can start using it in other places (rather than just for audio).